### PR TITLE
[skia] Fixing usage issues

### PR DESCRIPTION
**Pull request by dave-juicelabs**
**Describe the pull request**

- #### What does your PR fix?
  Fixes compilation when include SkColorSpace.h. It depends on modules/skcms/skcms.h which is not available in the installed includes.

- #### Which triplets are supported/not supported? Have you updated the [CI baseline](https://github.com/microsoft/vcpkg/blob/master/scripts/ci.baseline.txt)?
  all previous supported

- #### Does your PR follow the [maintainer guide](https://github.com/microsoft/vcpkg/blob/master/docs/maintainers/maintainer-guide.md)?
  Yes

- #### If you have added/updated a port: Have you run `./vcpkg x-add-version --all` and committed the result?
  Yes

**If you are still working on the PR, open it as a Draft: https://github.blog/2019-02-14-introducing-draft-pull-requests/**

### CODE REVIEW

**Review by dg0yt — CHANGES_REQUESTED** [review]
I don't think the `RelWithDebInfo` code belongs into a port because there is no such config. Consuming CMake projects can simply use [`CMAKE_MAP_IMPORTED_CONFIG_<CONFIG>`](https://cmake.org/cmake/help/latest/variable/CMAKE_MAP_IMPORTED_CONFIG_CONFIG.html).

**Comment by JackBoosY** [comment]
`RelWithDebInfo` should belong to triplet, not port.

**Comment by dave-juicelabs** [comment]
I have removed the RelWithDebInfo changes and left the rest of the commit.

**Review by LilyWangLL — APPROVED** [review]

**Comment by dave-juicelabs** [comment]
Any update on this PR?

**Comment by JackBoosY** [comment]
Waiting for further review.

File: ports/skia/portfile.cmake [hunk 1]
```diff
@@ -170,6 +170,11 @@ if("vulkan" IN_LIST FEATURES)
      list(APPEND SKIA_PUBLIC_DEFINITIONS SK_VULKAN)
  endif()
 
+checkout_in_path("${EXTERNALS}/vulkanmemoryallocator"
+    "https://chromium.googlesource.com/external/github.com/GPUOpen-LibrariesAndSDKs/VulkanMemoryAllocator"
+    "7de5cc00de50e71a3aab22dea52fbb7ff4efceb6"
+)
```
Comment (dg0yt) [inline ports/skia/portfile.cmake:176]: Is this needed when feature `vulkan` isn't used?

**Review by dg0yt — COMMENTED** [review]
There is also a new release, 0.37.0, which has changes around skcms.

File: ports/skia/portfile.cmake (hunk 1 above)
Comment (LilyWangLL) [reply ports/skia/portfile.cmake:176]: Ping @dave-juicelabs for response.

**Comment by BillyONeal** [comment]
Skia has changed substantially since you authored these changes and it isn't clear that the bug you are working around is still present. If it is, please resolve the merge conflicts and respond here.

Thanks for your contribution to vcpkg!

**Comment by LilyWangLL** [comment]
> Skia has changed substantially since you authored these changes and it isn't clear that the bug you are working around is still present. If it is, please resolve the merge conflicts and respond here.
> 
> Thanks for your contribution to vcpkg!

Pinging @dave-juicelabs for response.

**Comment by LilyWangLL** [comment]
Convert this PR to draft since there is no progress. Please ping us if this PR is ready for review again.

**Comment by LilyWangLL** [comment]
Closing this PR since it seems that no progress is being made. Please ping us to reopen if work is still being done.

**Pull request closed without merging**